src/late_multicellular_stage/editor/LateMulticellularEditor.cs

```diff
@@ -63,6 +64,7 @@ public class LateMulticellularEditor : EditorBase<EditorAction, MulticellularSta
     private Light bodyEditorLight = null!;
 
     private WorldEnvironment worldEnvironmentNode = null!;
+    private Environment? environment;
```

Comment:
I think this variable might need to be disposed, as if the environment is currently detached when this node is removed from the scene tree the environment will cause a resource leak as it won't be automatically disposed due to being unattached.